Warcraft.NET/Files/phys/Entries/JOINEntry.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.IO;
+using Warcraft.NET.Files.Structures;
+
+namespace Warcraft.NET.Files.phys.Entries
+{
+    
+
+    public class JOINEntry
+    {
+        public enum Joint_Type : ushort
```

Comment:
Flags and enum's can be moved out into a own file. Create a folder called Enums in the Phys folder and a file called JointType